doc/manual/metaprogramming.rst

```diff
@@ -415,11 +418,16 @@ Basics
 
 Here is an extraordinarily simple macro:
 
-.. doctest::
+.. testcode::
 
-    julia> macro sayhello()
-               return :( println("Hello, world!") )
-           end
+    macro sayhello()
+        return :( println("Hello, world!") )
+    end
+
+.. testoutput::
```

Comment:
so this runs as a doctest, checks the output matches but doesn't include the output in the rendered docs?